specification/data-resource-semantic-conventions.md

```diff
@@ -4,7 +4,7 @@ This document defines standard attributes for resources. These attributes are ty
 [OpenCensus Resource standard](https://github.com/census-instrumentation/opencensus-specs/blob/master/resource/StandardResources.md).
 
 * [Service](#service)
-* [Library](#library)
+* [Telemetry SDK](#telemetry-sdk)
```

Comment:
This affects the glossary PR (https://github.com/dynatrace-oss-contrib/opentelemetry-specification/pull/10). Please remind me to update it, if this gets merged.

Reply:
Will (hopefully) do!
Please remind me to update it in my upstream PR if the glossary is merged first. 🙂